lstmcpipe/config/pipeline_config.py

```diff
@@ -45,52 +47,36 @@ def load_config(config_path):
     config = parse_config_and_handle_global_vars(loaded_config)
 
     log.info(
-        f'************ - Configuration for processing of {config["prod_type"]} using the {config["workflow_kind"]} pipeline:- ************'
+        f'************ - lstMCpipe will be launch using the {config["workflow_kind"]} pipeline:- ************'
     )
-    if config.get("DL0_input_dir"):
+    log.info(f'\nPROD_ID to be used: {config["prod_id"]}')
+    log.info("\nStages to be run:\n - " + "\n - ".join(config["stages_to_run"]))
+
+    if "merge_dl1" in config["stages_to_run"]:
+        log.info("Merging options:" f"\n - No-image argument: {config['merge_dl1']['merging_no_image']}")
+
+    if "r0_to_dl1" in config["stages_to_run"]:
         log.info(
-            "Simtel DL0 files are going to be searched at: "
-            f'{config["DL0_input_dir"].format("{" + ",".join(config["all_particles"]) + "}")}'
+            f"Simtel DL0 files will be get from:" + "\n - ".join([i["input"] for i in config["r0_to_dl1"]])
         )
-    elif config.get("DL1_input_dir"):
+    elif "dl1ab" in config["stages_to_run"]:
+        log.info(f'\nApplying dl1ab processing to MC prod: {config["dl1_reference_id"]}')
         log.info(
-            "DL1 base files are going to be searched at: "
-            f'{config["DL1_input_dir"].format("{" + ",".join(config["all_particles"]) + "}")}'
+            f"\nDL1 .h5 files will be get from:" + "\n - ".join([i["input"] for i in config["dl1ab"]])
         )
+    else:
+        pass
```

Comment:
not necessary

Reply:
or shoud `raise NotImplementedError`?

Reply:
Just checking which is the first stage to be run. If it's `r0_dl1` it will show these paths, if it's `dl1ab`, it will show these others.
Otherwise it will just not show nothing ? Should it ? there are many cases to manage...